react/src/util/api-client.ts

```diff
@@ -1,14 +1,39 @@
 import axios from 'axios';
 
+// export const getAddressDetails = async (
+//   address: string,
+// ): Promise<AddressDetails> => {
+//   const res = await fetch(
+//     `https://rest.bitcoin.com/v2/address/details/${address}`,
+//   );
+//   return await res.json();
+// };
+
 export const getAddressDetails = async (
   address: string,
 ): Promise<AddressDetails> => {
   const res = await fetch(
-    `https://rest.bitcoin.com/v2/address/details/${address}`,
+    `https://api.paybutton.org/address/transactions/${address}`,
   );
```

Comment:
@Klakurka Should we change this url to be a parameter as well?

Reply:
Ya anything that uses the api should be I think.